src/ya-comiste-rust/auth/tests/google_id_token_integration.rs

```diff
@@ -45,17 +45,3 @@ fn it_returns_error_on_unsigned_token() {
         r#"InvalidToken("cannot obtain Google certificate for key ID: \'123\'")"# // 123 is from the JWT header
     );
 }
-
-#[test]
-fn it_returns_error_on_expired_token() {
```

Comment:
This was a flaky test - Google rotated the certs so it cannot work (cannot verify the token).